### PR TITLE
Incorporated "master pipeline script" introduction from bsmith89/make-bml

### PR DESCRIPTION
**Pull request by bsmith89**
Incorporated one change to the lesson tried out at an intermediate
bioinformatics workshop.
Specifically, added a "master script" which runs the full analysis
pipeline described in the introduction.
Also added additional description of how a script like this
still fails to help when you make changes to your analysis.

### CODE REVIEW

File: 01-intro.md [hunk 1]
```diff
@@ -80,15 +92,102 @@ Together these scripts implement a common workflow:
...
+Using your text editor of choice (e.g. nano), add the following to a file named
+`run_pipeline.sh`.
+
+~~~ {.bash}
+# USAGE: bash run_pipeline.sh
+# to produce plots for isles and abyss.
+
+python wordcount.py isles.txt isles.dat
+python wordcount.py abyss.txt abyss.dat
+
```
Comment (mikej888) [inline 01-intro.md]: isles.txt should be books/isles.txt and abyss.txt should be books/abyss.txt.

File: 01-intro.md [hunk 2]
```diff
@@ -80,15 +92,102 @@ Together these scripts implement a common workflow:
...
+3.  It prevents us from _repeating_ typos or mistakes.
+    You might not get it right the first time, but once you fix something
+    it'll stay fixed.
+
+Despite these benefits it has a few shortcomings.
+
+Let's adjust the width of the bars in our plot produced by `plotcount.py`.
+Edit the script so that the bars are 0.8 units wide instead of 1 unit.
+(Hint: replace `width = 1.0` with `width = 1.0` in the definition of
+`plot_word_counts`.)
```
Comment (mikej888) [inline 01-intro.md]: The latter 1.0 should be 0.8.

File: 01-intro.md [hunk 3]
```diff
@@ -80,15 +92,102 @@ Together these scripts implement a common workflow:
...
+Another popular option is to comment out a subset of the lines in
+`run_pipeline.sh`:
+
+~~~ {.bash}
+# USAGE: bash run_pipeline.sh
+# to produce plots for isles and abyss.
+
+#python wordcount.py isles.txt isles.dat
+#python wordcount.py abyss.txt abyss.dat
+
```
Comment (mikej888) [inline 01-intro.md]: isles.txt should be books/isles.txt and abyss.txt should be books/abyss.txt.

**Comment by mikej888** [comment]
Please address the comments then I'll merge. It is a good addition to the introduction and makes the motivations of Make much clearer :-) Thanks!

**Comment by bsmith89** [comment]
Good catche(s).  Sorry about that.

**Comment by mikej888** [comment]
Merged, thanks again :-)

**Pull request merged**